src/components/articles/CommentCard.jsx

```diff
@@ -0,0 +1,13 @@
+export default function CommentCard({ comment }) {
+  const time =
+    comment.created_at.slice(0, 10) + " " + comment.created_at.slice(11, 16);
```

Comment:
Interesting way to handle the formatting of the date! I also think the method .toUTCString() will be helpful to get similar formatting, without having to do it manually.